### DISCUSSION TITLE
Flymake Faces (in-buffer)

### DISCUSSION THREAD

**Discussion by gs-101** [pspiagicw/catppuccin-emacs#203] [Q&A] [answered]
![*Embark Export: describe-face - flymake* 08:02 AM 09-12-2024](https://github.com/user-attachments/assets/50c0e6c3-cbf1-41b5-abdd-0bbc3319aa5c)

Why are the Flymake faces set for descriptions and notes customized, but not the ones used in regions (that is, the ones found on the buffer)?

I'm asking this as I tried to configure them with Catppuccin's colors with `set-face-attribute`, but had no success.

![flymake el gz 08:06 AM 09-12-2024](https://github.com/user-attachments/assets/06b79f10-67d2-4b80-a899-fa8f559fc83a)
`flymake-error` uses the default value of "Red1", for example.

**Comment by jtbx** [comment, marked as answer]
#206
